packages/dd-trace/src/exporters/log/index.js

```diff
@@ -2,7 +2,7 @@
 
 const log = require('../../log')
 
-const MAX_SIZE = 255 * 1024 // 255kb
+const MAX_SIZE = 50 * 1024 // 50kb
```

Comment:
Why 50kb and not 64kb as per testing?

Reply:
I wanted to leave some head room in case, but checking my results again it looks like it is consistently cutting off at 2^16 + 1 bytes, almost 64 kb exactly, regardless of memory size.